packages/apputils-extension/schema/themes.json

```
@@ -105,9 +109,27 @@
     "theme": {
       "type": "string",
       "title": "Selected Theme",
-      "description": "Application-level visual styling theme",
+      "description": "Application-level visual styling theme. Ignored when Adaptive Theme is enabled.",
       "default": "JupyterLab Light"
     },
+    "adaptive-theme": {
+      "type": "boolean",
+      "title": "Adaptive theme",
+      "description": "Sync styling theme with system settings",
```

```suggestion
      "description": "Synchronise styling theme with system settings",
```

Let's make lives of translators easier.